crmd/control.c

```diff
@@ -957,7 +968,7 @@ pe_cluster_option crmd_opts[] = {
           "Delay cluster recovery for the configured interval to allow for additional/related events to occur.\n"
           "Useful if your configuration is sensitive to the order in which ping updates arrive."
         },
-	{ "stonith-watchdog-timeout", NULL, "time", NULL, NULL, &check_sbd_timeout,
+	{ "stonith-watchdog-timeout", NULL, "time", NULL, NULL, &verify_stonith_watchdog_timeout,
```

Comment:
I think we want to keep check_sbd_timeout here. This is just to validate that the given value has a legal syntax, not that it is usable. I'm not sure stonithd is guaranteed to be available when we do the validation.

Reply:
Yes probably not a good idea to check that here - as we've been doing so far.
Moving that further check to e.g. roughly where startup-fencing is happening would as well solve the issue of stonithd potentially not being available at that early stage.

Reply:
On startup this is the wrong place - definitely. (Right after te connects the stonith-api looks like a better place.)
But I guess on subsequent changes during operation we still want to bring the node down as quickly as possible when we find that the local watchdog-config doesn't match the new cluster-config.